astropy/time/core.py

```diff
@@ -418,12 +418,12 @@ def __init__(self, val, val2=None, format=None, scale=None,
                 # check the location can be broadcast to self's shape.
                 self.location = np.broadcast_to(self.location, self.shape,
                                                 subok=True)
-            except Exception:
+            except Exception as e:
```

Comment:
Generally prefer to avoid 1-letter variable names in new code.  The standard idiom in this module is `err`, but I also use `exc`.